robotcontainer.py

```diff
@@ -255,9 +254,15 @@ def configureButtonBindings(self):
             ShootBall(self.indexer)
         ).whenReleased(HoldBall(self.indexer))
 
-        commands2.button.JoystickButton(*self.operatorInterface.fenderShot).whileHeld(
+        commands2.button.JoystickButton(
+            *self.operatorInterface.automaticShoot
+        ).whileHeld(
```

Comment:
Maybe toggle or make it a default if we can make it work. Holding seems a little weird for the automatic shot

Reply:
just a pretty fix for now, default functionality can come later